corpus2alpino/annotators/enrich_lassy.py

```diff
@@ -0,0 +1,114 @@
+#!/usr/bin/env python3
+"""
+Allows enriching nodes of a Lassy annotation using a dictionary
+"""
+from typing import cast, Dict, List
+
+import csv
+from lxml import etree
+
+from corpus2alpino.abstracts import Annotator
+from corpus2alpino.annotators.alpino import ANNOTATION_KEY
+from corpus2alpino.log import LogSingleton
+from corpus2alpino.models import Document, Utterance
+
+
+class EnrichLassyAnnotator(Annotator):
+    def __init__(self, enrichment_file: str):
+        """Creates an annotator which will add additional attributes to
+        nodes when they match the properties specified in an enrichment
+        file.
+
+        Arguments:
+            enrichment_file {str} -- Path to a comma-separated enrichment file
+            matching attributes columns should be prefixed with @, assignments
+            without. For example the following CSV would match 
+            <node pos="noun" num="pl" /> and add penn_pos="NNS":
+
+            @pos,@num,penn_pos
+            noun,pl,NNS
+            noun,,NN
```

Comment:
Please document what empty cells do: does this row simply not care about the `num` attribute, or should a `num` attribute be absent, or something else?